src/main/java/uk/gov/pay/webhooks/webhook/resource/CreateWebhookRequest.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.pay.webhooks.webhook.resource;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public record CreateWebhookRequest(
+        @JsonProperty("service_id") @NotEmpty @Size(max = 30) String serviceId,
+        @JsonProperty("live") @NotNull Boolean live,
```

Comment:
should this be `boolean` i.e. do we want to allow null values?

Reply:
AFAIK you can't annotate primitive boolean because even though you don't want to allow null values - values _can_ be null